### DISCUSSION TITLE
Is there any option to use redis cache / memcache ?

### DISCUSSION THREAD

**Discussion by lajeeshk** [YetiForceCompany/YetiForceCRM#15356] [General]
I would like to use redis cache / memcache.
does it to support new cache drivers ?

**Comment by mariuszkrzaczkowski** [comment]
Yes, look here
https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/app/Cache
it's very easy to get another one

**Reply by mariuszkrzaczkowski** [reply]
create a pull request as you do the new driver
